src/node.rs

```diff
@@ -0,0 +1,28 @@
+#[derive(Clone, PartialEq, Debug)]
+pub enum Node {
+    Atom(Atom),
+    List(List),
+}
+
+pub type Atom = &'static str;
```

Comment:
Does `Atom` actually use static strings? Sounds like a job for String to me

Reply:
Because we are directly writing programs on the test case. So I didn't think much and choose static string for that.